variants/frontend-base/js-lint/template.rb

```diff
@@ -2,7 +2,8 @@
 # ######################################
 
 run "rm .browserslistrc"
-run "yarn add --dev eslint eslint-config-ackama eslint-plugin-node eslint-plugin-import eslint-plugin-prettier prettier prettier-config-ackama eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y"
+run "yarn add --dev eslint eslint-config-ackama eslint-plugin-node eslint-plugin-import eslint-plugin-prettier prettier prettier-config-ackama eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y jest eslint-plugin-testing-library eslint-plugin-jest-dom eslint-plugin-jest"
```

Comment:
🐘 I think this should be refactored to use an `w%` array, like I did with the [typescript varient](https://github.com/ackama/rails-template/pull/207/files#diff-23eb3fd5a078907cdb393b92e280dc3c885f44e38857baf6b6f6b81e63535576R6-R22).

I'm happy for this to be done in its own PR.

Reply:
```suggestion
run "yarn add --dev eslint eslint-config-ackama eslint-plugin-node eslint-plugin-import eslint-plugin-prettier prettier prettier-config-ackama eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y jest eslint-plugin-testing-library eslint-plugin-jest-dom eslint-plugin-jest eslint-plugin-jest-formatting"
```

(this'll be required when you add `ackama/jest`)